### ISSUE TITLE
Feature request: binary keys for maps

### ISSUE THREAD

**Issue by olivermt** [msaraiva/exsamples#1]
I am using this stuff to functionally test api endpoints, and keys there are always strings and not atoms.

Right now I have to Enum.into every sample I use.

**Comment by msaraiva** [comment]
It looks like a nice addition. I think we have two options:

1- Add an extra option to `samples`. Something like:

``` Elixir
    samples stringify_keys: true do
      :name       | :country        | :city
      "Christian" | "United States" | "New York City"
      "Peter"     | "Austria"       | "Vienna"
    end
```

2- Let define the first row as strings:

``` Elixir
    samples do
      "name"      | "country"       | "city"
      "Christian" | "United States" | "New York City"
      "Peter"     | "Austria"       | "Vienna"
    end
```

Still not sure which one would be better. I'll take a look at it.

**Comment by olivermt** [comment]
I think the first one is better in terms of backwards compability at least.

**Comment by OvermindDL1** [comment]
Second is best for consistency though, and I am not sure how it would not be backwards incompatible either?

**Comment by Malian** [comment]
I am voting for (2) for consistency as
```elixir
samples do
  %{}   | "name"      | "country"       | "city"
  user1 | "Christian" | "United States" | "New York City"
  user2 | "Ingrid"    |   "Austria"     | "Salzburg"
end
```
already returns string keys
```elixir
[
  %{"city" => "New York City", "country" => "United States", "name" => "Christian"},
  %{"city" => "Salzburg", "country" => "Austria", "name" => "Ingrid"}
]
```

I made a PR for those who want to test: https://github.com/msaraiva/exsamples/pull/5

**Comment by msaraiva** [comment]
Fixed in #5.
